logstash-core/logstash-core.gemspec

```diff
@@ -14,12 +14,11 @@ Gem::Specification.new do |gem|
   gem.files         = Dir.glob(["logstash-core.gemspec", "lib/**/*.rb", "spec/**/*.rb", "locales/*", "lib/logstash/api/init.ru"])
   gem.test_files    = gem.files.grep(%r{^(test|spec|features)/})
   gem.name          = "logstash-core"
-  gem.require_paths = ["lib"]
+  gem.require_paths = ["lib", "vendor/jars"]
```

Comment:
Can use the same directory as our other projects?

logstash-core-event-java put them in `lib/logstash-core-event-java/
Kafka (in|out) and beats put them in `vendor/jar-dependencies`